### PR TITLE
logout session done register page also modified

### PR DESCRIPTION
**Pull request by VINITCHAHAL**

### CODE REVIEW

File: cart/static/css/product.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.product-price {
+    color: #b12704; /* Amazon-like red for prices */
+    font-size: 18px; /* Slightly larger font size */
+    margin: 5px 0; /* Margin for spacing */
+}
+
+.product-quantity {
+    color: #555; /* Gray for quantity */
+    margin: 5px 0; /* Margin for spacing */
+}
```
Comment (tmeerr) [inline cart/static/css/product.css:41]: Remove comments

File: cart/static/css/register.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.social-btn.twitter {
+    background-color: #1da1f2; /* Twitter color */
+}
+
+.social-btn.linkedin {
+    background-color: #0077b5; /* LinkedIn color */
+}
+
+.social-btn:hover {
+    opacity: 0.8; /* Slight opacity change on hover */
```
Comment (tmeerr) [inline cart/static/css/register.css]: remove comments

File: cart/static/css/product.css (hunk 1 above)
Comment (VINITCHAHAL) [reply cart/static/css/product.css:41]: Ok Sir

File: cart/static/css/register.css (hunk 2 above)
Comment (VINITCHAHAL) [reply cart/static/css/register.css]: Ok Sir

**Pull request merged**